paloma-testnet-11/genesis.json

```diff
@@ -16322,7 +12220,7 @@
     "version": {}
   },
   "genesis_time": "2022-08-28T19:58:56.180953667Z",
-  "initial_height": "532372",
+  "initial_height": "1",
```

Comment:
Block height is referenced elsewhere in this file, I don't think we should reduce this.